### PR TITLE
feat: create more preset variables

### PR DESCRIPTION
**Pull request by sh7dm**
## Description
Create more preset variables

## Motivation and Context
Easier customization of framework for end user.

## TODO list:
- [x] `VApp`
- [x] `VAppBar`
- [x] `VAlert`
- [x] `VAutocomplete`
- [x] `VAvatar`
- [x] `VBadge`
- [x] `VBanner`
- [x] `VBottomNavigation`
- [x] `VBottomSheet`
- [x] `VBreadcrumbs`
- [x] `VBtn`
- [x] `VBtnToggle`
- [x] `VCalendar`
- [x] `VCard`
- [x] `VCarousel`
- [x] `VCheckbox`
- [x] `VChip`
- [x] `VChipGroup`
- [x] `VColorPicker`
- [x] `VContent`
- [x] `VCombobox`
- [x] `VCounter`
- [x] `VData`
- [x] `VDataIterator`
- [x] `VDataTable`
- [x] `VDatePicker`
- [x] `VDialog`
- [x] `VDivider`
- [x] `VExpansionPanel`
- [x] `VFileInput`
- [x] `VFooter`
- [x] `VForm`
- [x] `VGrid`
- [x] `VHover`
- [x] `VIcon`
- [x] `VImg`
- [x] `VInput`
- [x] `VItemGroup`
- [x] `VLabel`
- [x] `VLazy`
- [x] `VList`
- [x] `VMenu`
- [x] `VMessages`
- [x] `VNavigationDrawer`
- [x] `VOverflowBtn`
- [x] `VOverlay`
- [x] `VPagination`
- [x] `VSheet`
- [x] `VParallax`
- [x] `VPicker`
- [x] `VProgressCircular`
- [x] `VProgressLinear`
- [x] `VRadioGroup`
- [x] `VRangeSlider`
- [x] `VRating`
- [x] `VResponsive`
- [x] `VSelect`
- [x] `VSkeletonLoader`
- [x] `VSlider`
- [x] `VSlideGroup`
- [x] `VSnackbar`
- [x] `VSparkline`
- [x] `VSpeedDial`
- [x] `VStepper`
- [x] `VSubheader`
- [x] `VSwitch`
- [x] `VSystemBar`
- [x] `VTabs`
- [x] `VTextarea`
- [x] `VTextField`
- [x] `VTimeline`
- [x] `VTimePicker`
- [x] `VToolbar`
- [x] `VTooltip`
- [x] `VTreeview`
- [x] `VWindow`

## Types of changes
-  [ ] Bug fix (non-breaking change which fixes an issue)
-  [x] New feature (non-breaking change which adds functionality)
-  [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-  [x] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:
-  [x] The PR title is no longer than 64 characters.
-  [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and breaking changes).
-  [x] My code follows the code style of this project.
-  [ ] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)
-  [ ] I've added new examples to the kitchen (applies to new features and breaking changes in core library)

### CODE REVIEW

File: packages/vuetify/src/components/VAvatar/VAvatar.sass [hunk 1]
```diff
@@ -1,8 +1,8 @@
-@import '../../styles/styles.sass'
+@import './_variables.scss'
 
 .v-avatar
   align-items: center
-  border-radius: 50%
+  border-radius: $avatar-border-radius
```
Comment (jacekkarczmarczyk) [inline packages/vuetify/src/components/VAvatar/VAvatar.sass]: Use it also in skeleton loader's avatar

Comment (sh7dm) [reply packages/vuetify/src/components/VAvatar/VAvatar.sass]: Hi @jacekkarczmarczyk, ty for help! I'm going through framework using a TODO list, will ref your review there!

File: packages/vuetify/src/components/VToolbar/VToolbar.sass [hunk 2]
```diff
@@ -109,14 +109,14 @@
     white-space: nowrap
 
 .v-toolbar.v-toolbar--collapsed
-  max-width: 112px
+  max-width: $toolbar-collapsed-width
```
Comment (jacekkarczmarczyk) [inline packages/vuetify/src/components/VToolbar/VToolbar.sass]:
```suggestion
  max-width: $toolbar-collapsed-max-width
```

Comment (sh7dm) [reply packages/vuetify/src/components/VToolbar/VToolbar.sass]: thanks

**Pull request merged**